From 5c499964ee6ea75cd2f3009fdd6a1b89d7ef2e49 Mon Sep 17 00:00:00 2001
From: Lu Wei <luwei32@huawei.com>
Date: Thu, 6 Jul 2023 22:12:41 +0800
Subject: [PATCH 615/780] [Huawei] netfilter: nf_tables:: fix kabi breakage in
 struct nfnetlink_subsystem

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I7FF4R

--------------------------------

Fix kabi breakage in struct nfnetlink_subsystem.

Signed-off-by: Lu Wei <luwei32@huawei.com>
---
 include/linux/netfilter/nfnetlink.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/netfilter/nfnetlink.h b/include/linux/netfilter/nfnetlink.h
index c2bb1ff6f9db..d0548aa68ac4 100644
--- a/include/linux/netfilter/nfnetlink.h
+++ b/include/linux/netfilter/nfnetlink.h
@@ -42,6 +42,7 @@ struct nfnetlink_subsystem {
 	int (*commit)(struct net *net, struct sk_buff *skb);
 	int (*abort)(struct net *net, struct sk_buff *skb,
 		     enum nfnl_abort_action action);
+	KABI_DEPRECATE_FN(void, cleanup, struct net *net)
 	bool (*valid_genid)(struct net *net, u32 genid);
 
 	KABI_RESERVE(1)
-- 
2.17.1

